imagetest/test_suites/oslogin/oslogin_test.go

```diff
@@ -16,8 +16,12 @@ import (
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
-const testUsername = "sa_105020877179577573373"
-const testUUID = "3651018652"
+// const testUsername = "sa_105020877179577573373"
```

Comment:
Comments are a duplicate of the code below them. Is there any non-corp documentation for where these came from that we can link to publicly?